native/tests/src/test_persistence.cpp

```diff
@@ -2,6 +2,7 @@
 #include <mixpanel/mixpanel.hpp>
 #include <mixpanel/detail/persistence.hpp>
 #include <thread>
+#include <fstream>
```

Comment:
Wouldn't it be better to `#include <fstream>` on `<mixpanel/detail/persistence.hpp>` so we don't need to import it here as well?